webview/WebView2_spec.md

```diff
@@ -0,0 +1,247 @@
+# Background
+
+The WebView2 is the WinUI 3.0 version of the WebView control. It includes usage of the updated Microsoft Edge browser based on the Chromium web engine.
+<br> 
+<br>
+[Documentation for the WinUI 2.0 version of the WebView control](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.WebView)
+<br>
+<br>
+[Discussion of the core Microsoft Edge WebView2](https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview2)
+
+# Description
+
+The WebView is a control that allows for HTML content to be hosted within an application. It renders web content using the Microsoft Edge rendering engine.
+Use a WebView control to display richly formatted HTML content from a remote web server, dynamically generated code, or content files in your app. Rich content can also contain script code and communicate between the script and your app’s code.
+
+# Examples
+
+**NavigateToString:**
+The following code example demonstrates how to load local HTML into a WebView2 control.
+
+```
+myWebView2.NavigateToString("<html><body><h2>This is an HTML fragment</h2></body></html>”);
+```
+<br>
+
+**Source:**
+To set the initial content of the WebView2, set the Source property in XAML. The XAML parser automatically converts the string to a Uri.
+
+```
+<!-- Source file is on the web. -->
+<WebView2 x:Name="MyWebView_1" Source="http://www.contoso.com"/>
+
+<!-- Source file is in local storage. -->
+<WebView2 x:Name="MyWebView_2" Source="ms-appdata:///local/intro/welcome.html"/>
+
+<!-- Source file is in the app package. -->
+<WebView2 x:Name="MyWebView_3" Source="ms-appx-web:///help/about.html"/>
+
+<!-- no provided uri initialization -->
```

Comment:
Since this doesn't set the Source, it's funny to put it in the Source examples. Maybe combine this with NavigateToString?